roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -30,6 +30,23 @@
     - monitoring.exporters.snmp_exporter.with_generator is defined      
     - monitoring.exporters.snmp_exporter.with_generator
 
+- name: "firewalld █ Add services to firewall's {{ prometheus_server_firewall_zone | default('public') }} zone"
+  firewalld:
+    zone: "{{ prometheus_server_firewall_zone | default('public') }}"
+    service: "{{ item }}"
+    immediate: "yes"
+    permanent: "yes"
+    state: enabled
+  when:
+    - ansible_facts.os_family == "RedHat"
+    - ep_firewall | default(false) | bool
+  loop:
+    - 9090/tcp
```

Comment:
Can you use prometheus service instead of 9090/tcp?

```
# firewall-offline-cmd --info-service prometheus
prometheus
  ports: 9090/tcp
  protocols: 
  source-ports: 
  modules: 
  destination: 
  includes: 
  helpers: 
```
It makes the firewall configuration easier to read with `firewall-cmd --info-zone <zone>`.

Reply:
Right, replaced by prometheus service